src/plugins/workspace/public/components/workspace_menu/workspace_menu.tsx

```diff
@@ -67,111 +81,92 @@ export const WorkspaceMenu = ({ coreStart }: Props) => {
     setPopover(false);
   };
 
-  const workspaceToItem = (workspace: WorkspaceObject) => {
-    const workspaceURL = formatUrlWithWorkspaceId(
-      coreStart.application.getUrlForApp(WORKSPACE_DETAIL_APP_ID, {
+  const workspaceToItem = (workspace: WorkspaceObject, itemType: string) => {
+    const appId = navGroupsMap![getUseCase(workspace)]?.navLinks[0].id;
+    const useCaseURL = formatUrlWithWorkspaceId(
+      coreStart.application.getUrlForApp(appId, {
         absolute: false,
       }),
       workspace.id,
       coreStart.http.basePath
     );
-    const name =
-      currentWorkspace?.name === workspace.name ? (
-        <EuiText>
-          <strong>{workspace.name}</strong>
-        </EuiText>
-      ) : (
-        workspace.name
-      );
+
+    const workspaceName = truncate(workspace.name, { length: MAX_WORKSPACE_NAME_LENGTH });
```

Comment:
How about using `workspace.name.substring()` directly?

Reply:
UX hopes to implement a style with ellipses, such as covert `WorkspaceNameNameNameNameName` to `WorkspaceName...`

Reply:
Can we use the css way to do that?

Reply:
updated